iterator 용어 통일 및 제목 형식 수정 (#174)
yeonjuan/Typescript-Handbook-ko: iterators-and-generators.md

## 반복 가능 (Iterables)

[`Symbol.iterator`](https://www.typescriptlang.org/docs/handbook/symbols.html#symboliterator) 프로퍼티에 대한 구현이 있는 객체는 반복 가능한 것으로 간주합니다.
`Array`, `Map`, `Set`, `String`, `Int32Array`, `Uint32Array` 등과 같은 일부 내장된 타입에는 이미 `Symbol.iterator` 프로퍼티가 구현되어 있습니다.
객체의 `Symbol.iterator` 함수는 반복할 값 목록을 반환합니다.

> ### `for..of` 문 (`for..of` statements)

Reviewer: ```suggestion
## `for..of` 문 (`for..of` statements)
```

`for..of`는 반복 가능한 객체를 반복하여 객체의 `Symbol.iterator`프로퍼티를 호출합니다.

> 다음은 배열의 간단한 `for..of` 루프입니다.

Reviewer: ```suggestion
다음은 배열의 간단한 `for..of` 루프입니다.
```
```suggestion
다음은 배열의 간단한 `for..of` 루프입니다:
```

}
```

#### ESMAScript 2015 및 상위 버전 타케팅 (Targeting ECMAScript 2015 and higher)

> ECMAScipt 2015-호환 엔진을 타케팅하는 경우, 컴파일러는 엔진의 내장 반복자(ieterator) 구현을 대상으로하는 `for..of` 루프를 생성합니다.

Reviewer: ```suggestion
ECMAScipt 2015-호환 엔진을 타케팅하는 경우, 컴파일러는 엔진의 내장 반복자를 구현하는 `for..of` 루프를 생성합니다.
```
